client/auth/Login.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import { useRouter } from 'next/router';
+
+const Login = () => {
+  const router = useRouter();
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleLogin = () => {
+    
+    router.push('/');
+  };
+
+  return (
+    <div className="container">
+      <h1>Login</h1>
+      <form>
+        <label>
+          Email:
+          <input type="email" value={email} onChange={(e) => setEmail(e.target.value)} required />
+        </label>
+        <br />
+        <label>
+          Password:
+          <input type="password" value={password} onChange={(e) => setPassword(e.target.value)} required />
+        </label>
+        <br />
+        <button type="button" onClick={handleLogin}>
+          Login
+        </button>
+      </form>
+
+      <style jsx>{`
```

Comment:
Do not use this way to write styles, either write it in a separate files and import them as modules. Or use tailwind. 
Refer: [https://nextjs.org/docs/app/building-your-application/styling/css-modules](https://nextjs.org/docs/app/building-your-application/styling/css-modules)